linera-views-derive/src/lib.rs

```diff
@@ -374,52 +402,78 @@ pub mod tests {
         }
     }
 
+    #[derive(Clone)]
     pub struct SpecificContextInfo {
         attribute: Option<TokenStream2>,
         context: Type,
-        generics: TokenStream2,
+        generics: Option<AngleBracketedGenericArguments>,
+        where_clause: Option<TokenStream2>,
     }
 
     impl SpecificContextInfo {
         pub fn empty() -> Self {
             SpecificContextInfo {
                 attribute: None,
                 context: syn::parse_str("C").unwrap(),
-                generics: quote! { <C> },
+                generics: Some(parse_quote! { <C> }),
+                where_clause: None,
             }
         }
 
         pub fn new(context: &str) -> Self {
             SpecificContextInfo {
                 attribute: Some(quote! { #[view(context = #context)] }),
                 context: syn::parse_str(context).unwrap(),
-                generics: quote! {},
+                generics: None,
+                where_clause: None,
+            }
+        }
+
+        /// Sets the `where_clause` to a dummy value.
+        ///
+        /// Also adds a `MyParam` generic type parameter to the `generics` field.
```

Comment:
It would be nice to say why the dummy is needed and what MyParam is here for.

Reply:
Oops! I tried tweaking the documentation to make it a bit clearer. Thanks!